src/components/Welcome.js

```diff
@@ -0,0 +1,78 @@
+import {useState, useEffect} from 'react';
+
+export default function Welcome(){
+    const [userInfo, setUserInfo] = useState({})
+    const [isMobile, setIsMobile] = useState(false)
+
+    useEffect(()=>{
+        fetch('https://dev-assessment.netlify.app/.netlify/functions/me', {
+            method: 'GET',
+            headers: {
+                'Accept': 'application/json',
+                'Authorization': 'Bearer 1F8E9119293D34903371897CB1483FF66D25FDDC5D03C1BED6CC47510550B069'
+            }
+        })
+        .then(res => res.json())
+        .then(data => {setUserInfo(data)}) 
+    },[])
+
+    //checks for mobile window size
+    const handleResize = () => {
```

Comment:
💭 should use `media_queries` and not rely on `window` this is an anti-pattern in front-end development